app/src/components/market/sections/market_profile/market_status/closed.tsx

```diff
@@ -278,12 +281,33 @@ const Wrapper = (props: Props) => {
     }
   }, [provider, account, marketMakerAddress, marketMaker])
 
+  const getRealitioBalance = async () => {
+    if (!cpk || !account) {
+      return
+    }
+
+    // Check if user has reality balance to redeem
+    const realitioService = new RealitioService(
+      getContractAddress(networkId, 'realitio'),
+      getContractAddress(networkId, 'realitioScalarAdapter'),
+      provider,
+      account,
+    )
+
+    const claimable = await realitioService.getClaimableBond(question.id, cpk.address, question.currentAnswer)
+    const balance = await realitioService.getBalanceOf(cpk.address)
```

Comment:
Since `submitAnswer` previously was done without the CPK if !relay, then should we also be checking if the account address has a claimable bond to make it backwards compatible? Might need to add some logic for the claim to work directly though